.circleci/config.yml

```diff
@@ -22,25 +38,53 @@ jobs:
           command: |
             ./cc-test-reporter after-build --coverage-input-type lcov --exit-code $?
 
+jobs:
+  test-node-latest:
+    docker:
+      - image: circleci/node:latest
```

Comment:
Are you sure you want to test on latest? It's kind of a tag pointing somewhere and when they release 13 I think it's ok that tests do not pass; this might be preventing the releases.

Reply:
This is just running tests to let us know if things work or not, we do not need to require this job to pass on GitHub. It will not block anything unless we configure it to do so.

Reply:
Ah nice, I didn't know it was a possibility. Has it been already configured to be in such way? (not blocking the merges)